### ISSUE TITLE
How to change logo and background color

### ISSUE THREAD

**Issue by happycoding1106** [keplergl/kepler.gl#733]
I am very interested in Kepler.gl. I was trying to change logo into mine and background color. I'd appreciate it if you can let me know how to change logo and background color? Thanks

**Comment by happycoding1106** [comment]
for example, https://analytics.descarteslabs.com

**Comment by jok0436** [comment]
Assuming you mean the Website.

The Favicon can be found [here](https://github.com/keplergl/kepler.gl/blob/master/website/src/static/favicon.png)

Most Images/Tiles are loaded from [here](https://github.com/keplergl/kepler.gl/blob/master/website/src/content.js)

The Colors are changed from [here](https://github.com/keplergl/kepler.gl/blob/master/website/src/styles.js)

**Comment by heshan0131** [comment]
You can use dependency ingection to replace default Logo 

replace the `defaultProps.logoComponent` of `PanelHeader`

Use a different theme object to change the backgroundColor etc

**Comment by happycoding1106** [comment]
Thanks Shan.

**Comment by Spottybadrabbit** [comment]
I did try to change the default value for the Panel Header but no luck @heshan0131 it seems the Logo for Kepler.gl application is in src/components/common/logo.js. I found `Panel Header` in [base.js](https://github.com/keplergl/kepler.gl/blob/master/src/styles/base.js). Is this the right one? If not where is it located and what variable needs to be set for `customTheme` ?

There is a const named `LogoSvg` the `<g>` element is a child. Edit that or add a link to your svg by importing the react component.

`import { ReactComponent as Logo } from "./logo.svg";`
